### PR TITLE
Linked positions in OrgChart

### PR DESCRIPTION
**Pull request by ingridklepsvik**
added linked position  in orgchart

### CODE REVIEW

File: src/components/org/OrgChart/components/Links.tsx [hunk 1]
```diff
@@ -119,14 +120,20 @@ const Links = <T extends OrgStructure>() => {
                 return null;
             }
 
+            const linkClassnames = classNames(styles.link, {
+                [styles.isLinked]: node.linked,
+            });
+
+            console.log(node.linked);
```
Comment (eskil0312) [inline src/components/org/OrgChart/components/Links.tsx]: Remove console.log

File: src/components/icons/components/wysiwyg/LinkIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+
+import useIcon, { IconProps } from '../../../../hooks/useIcon';
+
+const LinkIcon = (props: IconProps) => {
+    const iconFactory = useIcon(
+        <path
+            fillRule="evenodd"
+            clipRule="evenodd"
+            d="M11 15H7C5.35 15 4 13.65 4 12C4 10.35 5.35 9 7 9H11V7H7C4.24 7 2 9.24 2 12C2 14.76 4.24 17 7 17H11V15ZM17 7H13V9H17C18.65 9 20 10.35 20 12C20 13.65 18.65 15 17 15H13V17H17C19.76 17 22 14.76 22 12C22 9.24 19.76 7 17 7ZM16 11H8V13H16V11Z"
```
Comment (eskil0312) [inline src/components/icons/components/wysiwyg/LinkIcon.tsx]: Is this file formatted?

File: src/components/org/OrgChart/OrgChart.stories.tsx [hunk 3]
```diff
@@ -4,6 +4,7 @@ import withFusionStory from '../../../../.storybook/withFusionStory';
 import OrgChart from '.';
 import { OrgStructure, OrgChartItemProps, BreadCrumb } from './orgChartTypes';
 import { useComponentDisplayType } from '@equinor/fusion';
+import { LinkIcon } from './../../../components/icons/components/wysiwyg/index';
```
Comment (eskil0312) [inline src/components/org/OrgChart/OrgChart.stories.tsx]: Could we change the import path to import { LinkIcon } from '@equinor/fusion'?

File: src/components/icons/components/wysiwyg/LinkIcon.tsx (hunk 2 above)
Comment (ingridklepsvik) [reply src/components/icons/components/wysiwyg/LinkIcon.tsx]: Yes

**Pull request merged**